digital-deck-server/api/controller.js

```diff
@@ -52,6 +57,23 @@ app.post("/new-session", function (req, res) {
   res.send(currSession);
 });
 
+router.post("/draw-cards", async function (req, res) {
```

Comment:
Why are we using POST instead of GET?

Reply:
Because it's updating the deck and player data in the backend also. I don't think GETs should be modifying any data.